src/Plugin/EntityBrowser/Display/Modal.php

```diff
@@ -216,20 +276,53 @@ public function widgetAjaxCallback(array &$form, FormStateInterface $form_state)
   }
 
   /**
-   * Helper function to return commands to return in AjaxResponse
+   * Helper function to return commands to return in AjaxResponse.
    *
    * @return array
+   *   An array of ajax commands.
    */
   public function getAjaxCommands(FormStateInterface $form_state) {
     $entities = array_map(function(EntityInterface $item) {return [$item->id(), $item->uuid(), $item->getEntityTypeId()];}, $form_state->get(['entity_browser', 'selected_entities']));
+
     $commands = array();
     $commands[] = new SelectEntitiesCommand($this->uuid, $entities);
+    // This doesn't work for now. Probably because we are in the iframe context.
```

Comment:
We should probably create new command that climbs up to the parent page and executes similar JS code there.